README.md

```diff
@@ -0,0 +1,219 @@
+
+    **Note:** ``bootstrap`` also runs ``check`` (see below) and fails if ``check`` fails.
+
+* ``check``:
+
+    Checks if the [yarn-lock.json](https://yarnpkg.com/lang/en/docs/yarn-lock/) and ``package.json`` files in the local package tree are in sync. This helps ensure your dependency versions are consistent across machines.
+    For example, running ``binge check`` in [./examples/check-sync/root](./examples/check-sync/root) returns:
+
+    ```
+    Failure
+    Lock out of sync
+    Node name: root
+    Node path: …/binge/examples/check-sync/root
+    'angular' wanted 1.6.5 but no match found on the lock file
+    ```
+
+*  ``copy``:
+
+    Copies a file into each node of the local package tree.
+    This is helpful to manage configuration file or credentials when preparing production releases.
+    For example, when creating the file ``.cred`` in [./examples/bootstrap](./examples/bootstrap), navigating to ``root`` and running ``binge copy ../.cred``, binge copies the file ``.cred`` to the following directories:
+    * ``i-filter-stuff``,
+    * ``i-print-stuff``,
+    * ``i-provide-colors``,
+    * ``root``.
+
+* ``graph``:
+
+    Prints the package tree and the layer topology (the build order of the nodes).
+    For example, running ``binge graph`` [./examples/bootstrap/root](./examples/bootstrap/root) returns:
+
+    ```
+    [Binge] Christmas Tree
+
+    root
+    ├── i-filter-stuff
+    └─┬ i-print-stuff
+      └── i-provide-colors
+
+    Layers:
+    1 root
+    2 i-print-stuff
+        3 i-filter-stuff
+        3 i-provide-colors
+    ```
+
+* ``harmony``:
+
+    Checks the local package tree for dependency consistency.
+    For example, ``binge harmony`` returns an *error* when two local packages contain incompatible version specifications of the same dependency…
+    ```
+    ------------------
+    |    q failed    |
+    ------------------
+    module-a ->  1.5.0
+    module-b ->  1.5.0
+    module-c ->  1.5.1
+    …
+    Error    q     failed -> module-a@1.5.0, module-b@1.5.0, module-c@1.5.1
+    ```
+    …and a *warning* when version specifications are *inconsistent*, but not *incompatible*:
+    ```
+    -------------------
+    |   fbjs 0.8.14   |
+    -------------------
+    module-a ->  0.8.14
+    module-b ->  0.8.14
+    module-c -> ^0.8.14
+    …
+    Warning  fbjs  0.8.14 -> module-a@0.8.14, module-b@0.8.14, module-c@^0.8.14
+    ```
+
+* ``install``:
+
+    Installs all reachable nodes in the local package tree to the ``node_modules`` folder of the global entry point. For example, running ``binge install`` in [./examples/bootstrap/root](./examples/bootstrap/root) installs the dependencies of the three local packages ``module-a``, ``module-b`` and ``module-c`` into the corresponding local ``node_modules`` folders, as well as into the ``node_modules`` folder of the global entry point ``root``.
+
```

Comment:
We could point out that all nodes get installed, with the hoisted version of its reachable package graph